nebula/ui/components/forms/VPNTextField.qml

```diff
@@ -38,7 +36,7 @@ TextField {
     leftPadding: VPNTheme.theme.windowMargin
     rightPadding: VPNTheme.theme.windowMargin
```

Comment:
Tiniest/non-blocking nit : let's increase `rightPadding` to something that takes the eye icon into account. Currently I think it's possible for there to be an overlap between the characters and the icon if the password is very long.

Reply:
Good point — increased `rightPadding`!